src/util/schnorr.rs

```diff
@@ -22,13 +22,12 @@ use hashes::{Hash, HashEngine};
 use util::taproot::{TapBranchHash, TapTweakHash};
 
 /// Untweaked Schnorr public key
-pub type UntweakedPublicKey = PublicKey;
```

Comment:
I tried using a type alias for `UntweakedPublicKey`, however, this would not let me implement methods onto it.

`cannot define inherent 'impl' for a type outside of the crate where the type is defined
define and implement a trait or new type instead`

If someone knows a work around it would be great.

Reply:
You will need to implement a `Tweakable` trait for it.